### PR TITLE
TailwindCSS setup.

### PR DESCRIPTION
**Pull request by manangouhari**
**What does this PR do?**
Setup tailwindcss with our design system colours. 

**How should this be manually tested?**
Use a tailwind class on any element. 

**Any background context you want to provide?**
Still need to setup typography and spacing.

### CODE REVIEW

**Comment by manangouhari** [comment]
@bhaskarSingh can you take a look at why the build is failing

**Comment by bhaskarSingh** [comment]
Look into this: https://github.com/tailwindlabs/tailwindcss/issues/2810#issuecomment-730258084 & https://tailwindcss.com/docs/upgrading-to-v2

`TypeError: getProcessedPlugins is not a function` is being thrown at build.

**Comment by manangouhari** [comment]
@bhaskarSingh I'm not able to replicate this error on my local setup. Requesting you to step in because I'm basically flying blind with this error, i.e., can't replicate.

**Comment by manangouhari** [comment]
@bhaskarSingh it's ready to be merged.

**Comment by bhaskarSingh** [comment]
The issue is fixed. Updated nelify node version to 12 since tailwind v2 requires node >= 12.

In-depth detail is here: https://tailwindcss.com/docs/upgrading-to-v2#upgrade-to-node-js-12-13-or-higher

**Comment by bhaskarSingh** [comment]
> @bhaskarSingh it's ready to be merged.

What is ?

**Comment by manangouhari** [comment]
This PR. Our timing was a little off, I wrote my comment a second before yours 😅
So didn't know you had fixed it.

File: tailwind.config.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
+const colors = require('tailwindcss/colors');
+
```
Comment (bhaskarSingh) [inline tailwind.config.js:2]: Does using prefilled values makes sense or should it be initialized with empty values just as stated here: https://tailwindcss.com/docs/guides/gatsby

Comment (manangouhari) [reply tailwind.config.js:2]: I've used values as defined by @harsh242 and Ayush in the Figma files.

Comment (bhaskarSingh) [reply tailwind.config.js:2]: Got it.

**Pull request merged**